gulpfile.js/tasks/init.js

```diff
@@ -2,7 +2,7 @@ const gulp = require('gulp');
 const colors = require('ansi-colors');
 const asciify = require('asciify');
 
-const task = (cb) => {
+const task = (done) => {
```

Comment:
I've renamed the callback everywhere to `done` because in gulp docs they also are using `done`:
https://github.com/gulpjs/gulp/blob/4.0/docs/API.md